src/operator/nn/layer_norm-inl.h

```diff
@@ -116,7 +116,7 @@ void LayerNormComputeGeneral(const nnvm::NodeAttrs& attrs,
   // Calculate mean
   MSHADOW_REAL_TYPE_SWITCH(outputs[0].type_flag_, DType, {
     BROADCAST_NDIM_SWITCH(red_dst_shape.ndim(), NDim, {
-      broadcast::Reduce<red::sum, NDim, DType, op::mshadow_op::identity>(
+      broadcast::Reduce<mshadow_op::sum, NDim, DType, mshadow_op::identity, true>(
```

Comment:
Should we read env var first before setting safe_acc to `True`?

Reply:
Fixed